### ISSUE TITLE
Windows HCM: Stepper

### ISSUE THREAD

**Issue by swisspost-devs** [swisspost/common-web-frontend#107]
The stepper does not display correctly in Windows High Contrast mode.
![image](https://user-images.githubusercontent.com/103635272/170206485-1e98744d-0feb-4c5b-8181-b9fab64c4ace.png)
![image](https://user-images.githubusercontent.com/103635272/170206746-86eccc71-215d-41c0-8fcb-325b52c22b15.png)

- [x] Missing line between steps
- [x] Active step is not clearly indicated
